will-kitchell/test/greet_test.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+var expect = require('chai').expect;
+var greet = require(__dirname + '/../lib/greet'); //must go back a directory, go into the lib directory, run greet.js
+
+describe('the greet function', function(){
+  it('should greet the user by name', function(){
+    expect(greet(process.argv[2])).to.eql('hello ' + process.argv[2]);
+  });
+});
```

Comment:
You should have separate tests for checking the function's output and checking that the process.argv arguments are being processed.